src/PresentationalComponents/Notification/notification.scss

```
@@ -0,0 +1,107 @@
+// js generates class '.page__{rootClass}' that wraps the content in your page added to the root tag
+// Found in Routes.js as the rootClass
+@import '~@redhat-cloud-services/frontend-components-utilities/styles/_mixins.scss';
+
+.pref-notification {
+  .pref-email__subs-info {
+    padding-bottom: 0;
+  }
+
+  .pf-c-data-list {
+    border: 0;
+
+    .pref-c-title {
+      // min and max width, because data lists
+      min-width: 155px;
+      max-width: 155px;
+    }
+    .pf-c-data-list__item:last-of-type { border: 0; }
+
+    .pf-c-data-list__item-content {
+      &.pref-u-condensed {
+        padding-top: 0;
+        padding-bottom: 0;
+      }
+
+      .pf-c-data-list__cell{
+        > * {
+          padding-bottom: var(--pf-global--spacer--md);
+
+          &:last-child {
+            padding-bottom: 0;
+          }
+        }
+      }
+    }
+
+    .pf-c-data-list__item-row { padding-left: 0; }
+    .pref-email__info-user-email {
+      padding-left: 19px;
```

This could be `--pf-global--spacer--lg`
```suggestion
      padding-left: var(--pf-global--spacer--lg);
```